packages/@expo/config-plugins/src/ios/Maps.ts

```diff
@@ -10,9 +10,8 @@ import { mergeContents, MergeResults, removeContents } from '../utils/generateCo
 
 const debug = require('debug')('expo:config-plugins:ios:maps') as typeof console.log;
 
-// Match against `UMModuleRegistryAdapter` (unimodules), and React Native without unimodules (Expo Modules), and SDK +44 React AppDelegate subscriber.
```

Comment:
afaik we don't need to match against any of these except what we include in the bare template